src/components/SignInPage/SignInSection.js

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from 'react';
+import { Row, Col, Form, Button } from 'react-bootstrap';
+
+function SignInSection() {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const handleEmail = (e) => {
+    setEmail(e.target.value);
+    console.log(email);
+  };
+  const handlePassword = (e) => {
+    setPassword(e.target.value);
+    console.log(password);
+  };
+
+  return (
+    <div className="d-flex align-items-center">
+      <div className="welcome-rectangle d-none d-md-flex align-items-center justify-content-center">
+        <h1>
+          WELCOME <br /> BACK
+        </h1>
+      </div>
+      <div className="signin-rectangle">
```

Comment:
use camelCase for classes naming.

Reply:
should i use camelCase or use the dash in class names 
(signInRectangle) or (sign-in-rectangle)

Reply:
@MustafaNawzat